frontend/src/dashboard/dashboard-route.tsx

```diff
@@ -18,7 +27,19 @@ const DashboardRoute = () => {
         <div>Jestem zalogowany</div>
       ) : (
         <div>
-          <Header>Nie jestem zalogowany</Header>
+          <TopicArea><h1>WELCOME TO TOURNAMENT4YOU</h1></TopicArea>
+          <hr></hr>
+          <TextArea>
```

Comment:
As fas as i know text area purpose is to provide user possibility to write some text, so it isn't probably right HTML tag

Reply:
The naming has been changed from TextArea to Text so that there should not be mismatches.